app/views/metrics/index.erb

```diff
@@ -0,0 +1,16 @@
+<div class="col-md-10">
+  <%= render 'shared/nav-filter' do |f|%>
+      <%= f.input :period,
+      wrapper_html: { class: 'period-selection' },
+      input_html: { class: 'margin-left' },
+      required: false,
+      collection: [['Daily', 'daily'], ['Weekly', 'weekly']],
+      selected: value_from_query_param(:period),
+      prompt: '- select option -'
+    %>
+  <% end %>
+  <%= render template: "projects/review_turnaround", locals: { review_turnaround: @review_turnaround } %>
+  <button type="button" class="btn btn-secondary details-button">View details</button>
```

Comment:
I have an alternative solution here:

We write a generic partial that contains:
- The project metric
- The button to see details
- The user-project metric

And we write it in a generic way, so that merge time (and other future metric) use it as well, passing the correct arguments each time it is rendered.

I'm open to opinions here

Reply:
I like it! I will put the metrics and the button on a review turnaround partial